zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/MappingController.java

```diff
@@ -87,13 +87,13 @@ public ResponseEntity<MappingItem> getMapping(@PathVariable final long mappingKe
       path = "/search",
       produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
       consumes = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity<MappingSearchQueryResponse> searchMappings(
+  public ResponseEntity<MappingSearchResponse> searchMappings(
```

Comment:
💭 It looked like `MappingSearchQueryResponse` was defined but not used in OpenAPI spec, while `MappingSearchResponse` was defined as response but not used here in controller.
Same with `MappingItem` & `MappingResponse`.

Reply:
Interesting. Might be something the Identity team mixed up accidentally.

Reply:
Probably. I guess the `MappingSearchQueryResponse` & `MappingItem` names are more consistent with our objects.
I'll switch the names back.